pkg/codefresh/argo_runtime.go

```diff
@@ -21,19 +22,65 @@ type (
 		}
 		Errors []graphqlError
 	}
+
+	graphQlRuntimeCreationResponse struct {
+		Data struct {
+			Runtime model.RuntimeCreationResponse
+		}
+		// TODO: Errors
+	}
 )
 
 func newArgoRuntimeAPI(codefresh *codefresh) IArgoRuntimeAPI {
 	return &argoRuntime{codefresh: codefresh}
 }
-func (r *argoRuntime) List() ([]model.Runtime, error) {
 
+func (r *argoRuntime) Create(runtimeName string) (*model.RuntimeCreationResponse, error) { // TODO: should also return error
+	type forJsonData interface{}
+	
+	// the newlines are necessary
+	var interpolatedMutation forJsonData = fmt.Sprintf("mutation {\n  runtime(name: \"%s\") {\n    id\n    newAccessToken\n  }\n}\n", runtimeName)
```

Comment:
the fact that it works only with the newlines (when sending an interpolated interface{} ), makes me think that this was the reason that it "suddenly" worked in the morning when sending the hard coded mutation string. A matter of breaking to a new line at the right spot.
(these newlines are taken from 'inspect' of playground like you suggested)